src/VirtualMonoRepo/InitializeVMR.proj

```diff
@@ -0,0 +1,121 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
+<Project>
+  <PropertyGroup>
+    <ArtifactsDir>$(RepoRoot)artifacts/</ArtifactsDir>
+    <VirtualMonoRepoTasksAssembly>$(ArtifactsDir)bin/VirtualMonoRepo.Tasks/$(Configuration)/net7.0/VirtualMonoRepo.Tasks.dll</VirtualMonoRepoTasksAssembly>
+
+    <VmrDir Condition=" '$(VmrDir)' != '' ">$([MSBuild]::EnsureTrailingSlash('$(VmrDir)'))</VmrDir>
+    <TmpDir Condition=" '$(TmpDir)' == '' ">$(RepoRoot)tmp/</TmpDir>
```

Comment:
If we can use `$(ArtifactsDir)tmp/` it should be gitignored already, otherwise I think we should add a gitignore for it.

Reply:
Ah, this should have been pointing to `artifacts/tmp` all along, thanks for noticing. Should be the same default as tarball has.